05/zviratka_zip.py

```diff
@@ -0,0 +1,11 @@
+pets = ['had', 'andulka', 'pes', 'kočka', 'králík']
+pets_start_second = []
+
+for pet in pets:
+    pets_start_second.append(pet[1:]) 
+pets_sorted_second = zip(pets_start_second, pets)
```

Comment:
Teoreticky, v tomto momente by sa ten zoznam nemal volať pets_storted_second, pretože ešte zoradený nie je i keď to s ním zamýšľaš. V zložitejšom kóde takýto zápis môže byť mätúci.